DOL.WHD.Section14c.Common/Extensions/StringExtension.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Text.RegularExpressions;
+
+namespace DOL.WHD.Section14c.Common.Extensions
+{
+    /// <summary>
+    /// Cistom string extension
+    /// </summary>
+    public static class StringExtension
+    {
+        /// <summary>
+        /// Trim string and convert to lower case
+        /// </summary>
+        /// <param name="value">String to convert</param>
+        /// <returns>Convert value</returns>
+        public static string TrimAndToLowerCase(this string value)
```

Comment:
I think it would be useful for this summary to note that this method also converts whitespace into single spaces.